src/ray/core_worker/core_worker.cc

```diff
@@ -273,6 +286,14 @@ CoreWorker &CoreWorkerProcess::GetCoreWorker() {
   EnsureInitialized();
   if (core_worker_process->options_.num_workers == 1) {
     auto global_worker = core_worker_process->GetGlobalWorker();
+    if (core_worker_process->CreateGlobalWorkerOnConstruction() && !global_worker) {
+      // This could only happen when the worker has already been shutdown.
+      // In this case, we should exit without crashing.
+      // TODO (scv119): A better solution could be returning error code
+      // and handling it at language frontend.
+      RAY_LOG(ERROR) << "The global_worker has already been shut down.";
```

Comment:
Maybe improve the error message? "The global worker has already been shutdown. This happens when the Python code accesses the Ray's worker after it is shutdown. The process will exit"?